config/routes.rb

```diff
@@ -409,6 +409,6 @@
 
   # Deprecated Links
   get 'teams-committees' => redirect('teams-committees-councils')
-  get 'panel/delegate-crash-course' => redirect('panel/delegate#delegate-crash-course')
+  get 'panel/delegate-handbook' => redirect('panel/delegate#delegate-handbook')
```

Comment:
Only the RHS needs to be changed. The LHS is actually meant to be the link that existed earlier and doesn't exist now, but still supporting in case the users have them bookmarked.